pkg/auth/event/event.go

```diff
@@ -37,7 +37,7 @@ type Event struct {
 
 func newEvent(seqNo int64, payload Payload, context Context) *Event {
 	return &Event{
-		ID:      uuid.New(),
+		ID:      fmt.Sprintf("%08x", seqNo),
```

Comment:
Why do we need to pad leading zeros?

Reply:
- All ID would have same length
- Can be indexed lexically with same order as sequence number

Reply:
```typescript
Math.pow(2, 64) === 18446744073709552000;
"18446744073709552000".length === 20;
```

Should we make width 20 instead of 8?

Reply:
It should be 16 characters, hex encoding is used.

Reply:
Why not just use decimal?

Reply:
It's shorter.

Reply:
It's shorter.